packages/web/src/components/gcds-radio/gcds-radio.stories.tsx

```diff
@@ -68,18 +68,18 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-radio
   radio-id="${args.radioId}"
   label="${args.label}"
```

Comment:
Same comment about the label: would you mind marking it as required in the story and setting a default type?